content/docs/operating/getting_started/tr1d1um.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Tr1d1um
+sort_rank: 5
+---
+
+# Tr1d1um
+
+# Installation
+-   [RPM](https://xmidt.io/download/#tr1d1um)
+-   [Binary](https://xmidt.io/download/#tr1d1um)
+-   Docker (Link TBC)
+
+# Configuration
+Refer to [configuration file](https://github.com/xmidt-org/tr1d1um/blob/master/tr1d1um.yaml)
+for how to configure tr1d1um.
+
+## Connecting to scytale
+Under the fanout block of the yaml file you will have
+
+```yaml
+targetURL: http://SCYTALE_HOSTNAME:PRIMARY_PORT
+```
+Where HOSTNAME is you DNS record or docker container or ip address.
```

Comment:
- [ ] `HOSTNAME` doesn't match `SCYTALE_HOSTNAME`
- [ ] Add explanation of `PRIMARY_PORT`?